pkg/kube2pulumi/kube2pulumi_test.go

```diff
@@ -0,0 +1,333 @@
+package kube2pulumi
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// PYTHON CODEGEN TESTS
+
+func TestNamespacePy(t *testing.T) {
+	assertion := assert.New(t)
+
+	pyExpected := `import pulumi
+import pulumi_kubernetes as kubernetes
+
+foo_namespace = kubernetes.core.v1.Namespace("fooNamespace",
+    api_version="v1",
+    kind="Namespace",
+    metadata={
+        "name": "foo",
+    })
+`
+	path, err := Kube2PulumiFile("../../testdata/Namespace.yaml", "python")
```

Comment:
Should use `filepath.Join` so that separators are platform independent.
```suggestion
	path, err := Kube2PulumiFile(filepath.Join("..", "..", "testdata", "Namespace.yaml"), "python")
```